openmc/mgxs/groups.py

```diff
@@ -77,6 +77,69 @@ def group_edges(self, edges):
         cv.check_greater_than('number of group edges', len(edges), 1)
         self._group_edges = np.array(edges)
 
+    @staticmethod
+    def get_energy_groups(name):
+        """Returns one of many commonly used and energy group structures.
+
+        Parameters
+        ----------
+        name : str
+            The name of the energy group structure. Pre-defined structures
+            include "CASMO-X" (where X is 2, 4, 8, 16, 25, 40 or 70).
```

Comment:
Is there a reference you can cite for the CASMO structures?

Reply:
Unfortunately, there is not a reference :-( I had a CASMO user plug in a special card into his CASMO input file in order to output the group structures. They are all derived from WIMS-69. I've added in a reference to the CASMO code itself; let me know if you like that, want it removed, or have another idea altogether.